### ISSUE TITLE
Persisting issue after update

### ISSUE THREAD

**Issue by Beauford20** [sbtoonz/BackPacks_Remake#59]
#57 

I installed the new .dll to my server but the trash bar remains below the trashcan on the inventory but now its not interactable now and is just on the UI.

**Comment by sbtoonz** [comment]
Do you happen to play the game in another language than english?

**Comment by Beauford20** [comment]
I do not, only english

**Comment by sbtoonz** [comment]
[https://youtu.be/qhAF3cKSdIE](url)

**Comment by Beauford20** [comment]
yeah mine does not say that

https://gyazo.com/fa0353c4146143fe964bce949df49414

**Comment by sbtoonz** [comment]
Can you please upload the Logoutput.log from the Bepinex folder?

**Comment by Beauford20** [comment]
[LogOutput.log](https://github.com/sbtoonz/BackPacks_Remake/files/9427570/LogOutput.log)

**Comment by Beauford20** [comment]
im actually reinstalling my entire server mod by mod atm

**Comment by sbtoonz** [comment]
can you do me a huge favor and open your bepinex.cfg file in the config folder

find the 
`[Harmony.Logger]` section and set its 
`LogChannels =`  to `All`

Next find 
`LogLevels =` and set it to `All`


then find the 
`[Logging.Disk]` section

`WriteUnityLog =` should be `true`
`Enabled =` should be `true`
`LogLevels =` set to `All`

Then I can get more data from the LogOuptut.log

**Comment by Beauford20** [comment]
yes, give me a few minutes to get the basic stuff back I had.

**Comment by Beauford20** [comment]
what in game manager do you recommend for editing configs in game, never used one before
[LogOutput.log](https://github.com/sbtoonz/BackPacks_Remake/files/9427696/LogOutput.log)
?

**Comment by sbtoonz** [comment]
https://github.com/BepInEx/BepInEx.ConfigurationManager/releases

The one by bepinex works great for me

**Comment by Beauford20** [comment]
yeah, im not sure how your shows that bar with the what key to press, not sure how you got it like that

**Comment by Beauford20** [comment]
I dropped the .4.12 dll in the plugin folder and still no joy.
